adr/README.md

```diff
@@ -0,0 +1,20 @@
+# Architecture Decision Records Folder
```

Comment:
I thought we were going to implement a top-level directory (i.e. /design) under which we'd have an adr directory.  @tonyespy suggested this as a way to group design documents together that might not be ADRs (e.g. legacy documents might fall under a /design/legacy directory structure).

Reply:
yes - will add/change